src/screens/toDoList/components/ToDoListRow.tsx

```diff
@@ -0,0 +1,127 @@
+import React, {memo, useEffect, useRef} from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  TouchableOpacity,
+  Dimensions,
+  Animated,
+  PanResponder,
+  Image,
+} from 'react-native';
+
+const SCREEN_WIDTH = Dimensions.get('window').width;
```

Comment:
If that's a constant - please move it to a constants file

Reply:
move to constant file